docs/file-upload-process.md

```diff
@@ -0,0 +1,107 @@
+## FILE UPLOAD PROCESS
+Below describes how the “file-upload” process works. The process described below is recommended if the client aims to upload multiple file (clients that aim to only upload a single file and are not interested in routing can follow a different [process](#single-file-process)): -
+
+### LIFE-CYCLE OF AN ENVELOPE - [see state diagram](../resources/images/envelope-life-cycle.png)
+
+#### 1. Create an envelope
+The first step is to create an envelope. The following endpoint is called and the envelope is created (an envelopeId is generated and returned in the responseHeader) : -
+
+- POST       /file-upload/envelopes
```

Comment:
You might need to point out that some calls such as this call it is a backend call : {{be-url}}/file-upload/envelopes